kc: new package at 0.0.6

mcpcpc opened the pull request:
Please fill out the template.

Read: https://k1ss.org/wiki/kiss/style-guide

---

## Description of package

KISS colors, a terminal-emulator color palette setter written in C and designed for KISS Linux.

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```

/var/db/kiss/installed/kc/version
/var/db/kiss/installed/kc/sources
/var/db/kiss/installed/kc/manifest
/var/db/kiss/installed/kc/etcsums
/var/db/kiss/installed/kc/depends
/var/db/kiss/installed/kc/checksums
/var/db/kiss/installed/kc/build
/var/db/kiss/installed/kc/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/kc
/usr/bin/
/usr/
/usr/share/palettes/light/tempus_totus
/usr/share/palettes/light/tempus_past
/usr/share/palettes/light/tempus_fugit
/usr/share/palettes/light/solarized
/usr/share/palettes/light/sexy-mostly-bright
/usr/share/palettes/light/github
/usr/share/palettes/light/base16-unikitty
/usr/share/palettes/light/base16-tomorrow
/usr/share/palettes/light/base16-summerfruit
/usr/share/palettes/light/base16-solarized
/usr/share/palettes/light/base16-shapeshifter
/usr/share/palettes/light/base16-one
/usr/share/palettes/light/base16-mexico
/usr/share/palettes/light/base16-materialer
/usr/share/palettes/light/base16-harmonic
/usr/share/palettes/light/base16-gruvbox-soft
/usr/share/palettes/light/base16-gruvbox-medium
/usr/share/palettes/light/base16-gruvbox-hard
/usr/share/palettes/light/base16-grayscale
/usr/share/palettes/light/base16-google
/usr/share/palettes/light/base16-github
/usr/share/palettes/light/base16-default
/usr/share/palettes/light/base16-cupertino
/usr/share/palettes/light/base16-cupcake
/usr/share/palettes/light/base16-classic
/usr/share/palettes/light/base16-atelier-sulphurpool
/usr/share/palettes/light/base16-atelier-seaside
/usr/share/palettes/light/base16-atelier-savanna
/usr/share/palettes/light/base16-atelier-plateau
/usr/share/palettes/light/base16-atelier-lakeside
/usr/share/palettes/light/base16-atelier-heath
/usr/share/palettes/light/base16-atelier-forest
/usr/share/palettes/light/base16-atelier-estuary
/usr/share/palettes/light/base16-atelier-dune
/usr/share/palettes/light/base16-atelier-cave
/usr/share/palettes/light/ashes
/usr/share/palettes/light/3024
/usr/share/palettes/light/
/usr/share/palettes/dark/zenburn
/usr/share/palettes/dark/vscode
/usr/share/palettes/dark/tempus_winter
/usr/share/palettes/dark/tempus_warp
/usr/share/palettes/dark/tempus_summer
/usr/share/palettes/dark/tempus_spring
/usr/share/palettes/dark/tempus_rift
/usr/share/palettes/dark/tempus_future
/usr/share/palettes/dark/tempus_dusk
/usr/share/palettes/dark/tempus_dawn
/usr/share/palettes/dark/tempus_autumn
/usr/share/palettes/dark/solarized
/usr/share/palettes/dark/sexy-zenburn
/usr/share/palettes/dark/sexy-x-dotshare
/usr/share/palettes/dark/sexy-visibone-alt-2
/usr/share/palettes/dark/sexy-visibone
/usr/share/palettes/dark/sexy-vacuous2
/usr/share/palettes/dark/sexy-user-77-mashup-colors
/usr/share/palettes/dark/sexy-trim-yer-beard
/usr/share/palettes/dark/sexy-tlh
/usr/share/palettes/dark/sexy-thwump
/usr/share/palettes/dark/sexy-theme2
/usr/share/palettes/dark/sexy-tartan
/usr/share/palettes/dark/sexy-tangoesque
/usr/share/palettes/dark/sexy-tango
/usr/share/palettes/dark/sexy-sweetlove
/usr/share/palettes/dark/sexy-swayr
/usr/share/palettes/dark/sexy-splurge
/usr/share/palettes/dark/sexy-simple_rainbow
/usr/share/palettes/dark/sexy-sexcolors
/usr/share/palettes/dark/sexy-s3r0-modified
/usr/share/palettes/dark/sexy-rydgel
/usr/share/palettes/dark/sexy-rezza
/usr/share/palettes/dark/sexy-rasi
/usr/share/palettes/dark/sexy-pretty-and-pastel
/usr/share/palettes/dark/sexy-phrak1
/usr/share/palettes/dark/sexy-parker_brothers
/usr/share/palettes/dark/sexy-orangish
/usr/share/palettes/dark/sexy-numixdarkest
/usr/share/palettes/dark/sexy-neon
/usr/share/palettes/dark/sexy-navy-and-ivory
/usr/share/palettes/dark/sexy-nancy
/usr/share/palettes/dark/sexy-muse
/usr/share/palettes/dark/sexy-monokai
/usr/share/palettes/dark/sexy-mikazuki
/usr/share/palettes/dark/sexy-mikado
/usr/share/palettes/dark/sexy-material
/usr/share/palettes/dark/sexy-kasugano
/usr/share/palettes/dark/sexy-jasonwryan
/usr/share/palettes/dark/sexy-invisibone
/usr/share/palettes/dark/sexy-insignificato
/usr/share/palettes/dark/sexy-hybrid
/usr/share/palettes/dark/sexy-hund
/usr/share/palettes/dark/sexy-gslob-nature-suede
/usr/share/palettes/dark/sexy-gotham
/usr/share/palettes/dark/sexy-gnometerm
/usr/share/palettes/dark/sexy-gjm
/usr/share/palettes/dark/sexy-euphrasia
/usr/share/palettes/dark/sexy-eqie6
/usr/share/palettes/dark/sexy-dwmrob
/usr/share/palettes/dark/sexy-dotshare
/usr/share/palettes/dark/sexy-doomicideocean
/usr/share/palettes/dark/sexy-digerati
/usr/share/palettes/dark/sexy-derp
/usr/share/palettes/dark/sexy-deafened
/usr/share/palettes/dark/sexy-dawn
/usr/share/palettes/dark/sexy-colorfulcolors
/usr/share/palettes/dark/sexy-cloud
/usr/share/palettes/dark/sexy-bitmute
/usr/share/palettes/dark/sexy-belge
/usr/share/palettes/dark/sexy-astromouse
/usr/share/palettes/dark/monokai
/usr/share/palettes/dark/hybrid-material
/usr/share/palettes/dark/gruvbox
/usr/share/palettes/dark/dkeg-wintry
/usr/share/palettes/dark/dkeg-view
/usr/share/palettes/dark/dkeg-victory
/usr/share/palettes/dark/dkeg-vans
/usr/share/palettes/dark/dkeg-urban
/usr/share/palettes/dark/dkeg-transposet
/usr/share/palettes/dark/dkeg-traffic
/usr/share/palettes/dark/dkeg-tealights
/usr/share/palettes/dark/dkeg-sundr
/usr/share/palettes/dark/dkeg-subtle
/usr/share/palettes/dark/dkeg-stv
/usr/share/palettes/dark/dkeg-squares
/usr/share/palettes/dark/dkeg-sprout
/usr/share/palettes/dark/dkeg-spire
/usr/share/palettes/dark/dkeg-soundwave
/usr/share/palettes/dark/dkeg-slate
/usr/share/palettes/dark/dkeg-skigh
/usr/share/palettes/dark/dkeg-simplicity
/usr/share/palettes/dark/dkeg-shade
/usr/share/palettes/dark/dkeg-scape
/usr/share/palettes/dark/dkeg-scag
/usr/share/palettes/dark/dkeg-relax
/usr/share/palettes/dark/dkeg-raild
/usr/share/palettes/dark/dkeg-provrb
/usr/share/palettes/dark/dkeg-prevail
/usr/share/palettes/dark/dkeg-poly
/usr/share/palettes/dark/dkeg-petal
/usr/share/palettes/dark/dkeg-pastely
/usr/share/palettes/dark/dkeg-parkd
/usr/share/palettes/dark/dkeg-paints
/usr/share/palettes/dark/dkeg-owl
/usr/share/palettes/dark/dkeg-novmbr
/usr/share/palettes/dark/dkeg-mattd
/usr/share/palettes/dark/dkeg-link
/usr/share/palettes/dark/dkeg-leaf
/usr/share/palettes/dark/dkeg-kit
/usr/share/palettes/dark/dkeg-harbing
/usr/share/palettes/dark/dkeg-fury
/usr/share/palettes/dark/dkeg-forst
/usr/share/palettes/dark/dkeg-flapr
/usr/share/palettes/dark/dkeg-fendr
/usr/share/palettes/dark/dkeg-escen
/usr/share/palettes/dark/dkeg-diner
/usr/share/palettes/dark/dkeg-designr
/usr/share/palettes/dark/dkeg-depth
/usr/share/palettes/dark/dkeg-corduroy
/usr/share/palettes/dark/dkeg-coco
/usr/share/palettes/dark/dkeg-chaires
/usr/share/palettes/dark/dkeg-bulb
/usr/share/palettes/dark/dkeg-brownstone
/usr/share/palettes/dark/dkeg-branch
/usr/share/palettes/dark/dkeg-book
/usr/share/palettes/dark/dkeg-blumune
/usr/share/palettes/dark/dkeg-bluetype
/usr/share/palettes/dark/dkeg-blok
/usr/share/palettes/dark/dkeg-blend
/usr/share/palettes/dark/dkeg-bark
/usr/share/palettes/dark/dkeg-amiox
/usr/share/palettes/dark/dkeg-5725
/usr/share/palettes/dark/darktooth
/usr/share/palettes/dark/base16tooth
/usr/share/palettes/dark/base16-zenburn
/usr/share/palettes/dark/base16-xcode-dusk
/usr/share/palettes/dark/base16-woodland
/usr/share/palettes/dark/base16-unikitty
/usr/share/palettes/dark/base16-twilight
/usr/share/palettes/dark/base16-tube
/usr/share/palettes/dark/base16-tomorrow-night
/usr/share/palettes/dark/base16-summerfruit
/usr/share/palettes/dark/base16-spacemacs
/usr/share/palettes/dark/base16-solarized
/usr/share/palettes/dark/base16-solarflare
/usr/share/palettes/dark/base16-snazzy
/usr/share/palettes/dark/base16-seti
/usr/share/palettes/dark/base16-rebecca
/usr/share/palettes/dark/base16-railscasts
/usr/share/palettes/dark/base16-porple
/usr/share/palettes/dark/base16-pop
/usr/share/palettes/dark/base16-pico
/usr/share/palettes/dark/base16-phd
/usr/share/palettes/dark/base16-paraiso
/usr/share/palettes/dark/base16-outrun
/usr/share/palettes/dark/base16-onedark
/usr/share/palettes/dark/base16-oceanicnext
/usr/share/palettes/dark/base16-ocean
/usr/share/palettes/dark/base16-nord
/usr/share/palettes/dark/base16-monokai
/usr/share/palettes/dark/base16-mocha
/usr/share/palettes/dark/base16-mellow-purple
/usr/share/palettes/dark/base16-materialer
/usr/share/palettes/dark/base16-material-palenight
/usr/share/palettes/dark/base16-material
/usr/share/palettes/dark/base16-materia
/usr/share/palettes/dark/base16-marrakesh
/usr/share/palettes/dark/base16-macintosh
/usr/share/palettes/dark/base16-isotope
/usr/share/palettes/dark/base16-irblack
/usr/share/palettes/dark/base16-icy
/usr/share/palettes/dark/base16-hopscotch
/usr/share/palettes/dark/base16-harmonic
/usr/share/palettes/dark/base16-gruvbox-soft
/usr/share/palettes/dark/base16-gruvbox-pale
/usr/share/palettes/dark/base16-gruvbox-medium
/usr/share/palettes/dark/base16-gruvbox-hard
/usr/share/palettes/dark/base16-greenscreen
/usr/share/palettes/dark/base16-grayscale
/usr/share/palettes/dark/base16-google
/usr/share/palettes/dark/base16-flat
/usr/share/palettes/dark/base16-embers
/usr/share/palettes/dark/base16-eighties
/usr/share/palettes/dark/base16-dracula
/usr/share/palettes/dark/base16-default
/usr/share/palettes/dark/base16-codeschool
/usr/share/palettes/dark/base16-classic
/usr/share/palettes/dark/base16-circus
/usr/share/palettes/dark/base16-chalk
/usr/share/palettes/dark/base16-brushtrees
/usr/share/palettes/dark/base16-bright
/usr/share/palettes/dark/base16-brewer
/usr/share/palettes/dark/base16-black-metal-venom
/usr/share/palettes/dark/base16-black-metal-nile
/usr/share/palettes/dark/base16-black-metal-mayhem
/usr/share/palettes/dark/base16-black-metal-marduk
/usr/share/palettes/dark/base16-black-metal-khold
/usr/share/palettes/dark/base16-black-metal-immortal
/usr/share/palettes/dark/base16-black-metal-gorgoroth
/usr/share/palettes/dark/base16-black-metal-funeral
/usr/share/palettes/dark/base16-black-metal-burzum
/usr/share/palettes/dark/base16-black-metal-bathory
/usr/share/palettes/dark/base16-black-metal
/usr/share/palettes/dark/base16-bespin
/usr/share/palettes/dark/base16-atelier-sulphurpool
/usr/share/palettes/dark/base16-atelier-seaside
/usr/share/palettes/dark/base16-atelier-savanna
/usr/share/palettes/dark/base16-atelier-plateau
/usr/share/palettes/dark/base16-atelier-lakeside
/usr/share/palettes/dark/base16-atelier-heath
/usr/share/palettes/dark/base16-atelier-forest
/usr/share/palettes/dark/base16-atelier-estuary
/usr/share/palettes/dark/base16-atelier-dune
/usr/share/palettes/dark/base16-atelier-cave
/usr/share/palettes/dark/base16-ashes
/usr/share/palettes/dark/base16-apathy
/usr/share/palettes/dark/base16-3024
/usr/share/palettes/dark/ashes
/usr/share/palettes/dark/3024
/usr/share/palettes/dark/
/usr/share/palettes/
/usr/share/
/etc/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

dylanaraps commented:
I think it's better for default colors to go in `/usr/share` as `/etc` is handled differently by the package manager.

mcpcpc commented:
> I think it's better for default colors to go in `/usr/share` as `/etc` is handled differently by the package manager.

Fixed upstream. Bumped version to 0.0.6, corrected the manifest and changed PR title.

dylanaraps commented:
Awesome. thanks.
